pkg/framework/fetcher.go

```diff
@@ -21,16 +21,13 @@ import (
 	"k8s.io/apimachinery/pkg/api/errors"
 	"k8s.io/apimachinery/pkg/types"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	"github.com/m88i/nexus-operator/pkg/logger"
 )
 
 func Fetch(client client.Client, key types.NamespacedName, instance resource.KubernetesResource, kind string) error {
-	log := logger.GetLoggerWithNamespacedName("resource_fetcher", key)
-	log.Debugf("Attempting to fetch deployed %s", kind)
+	log.Info("Attempting to fetch", "deployed", kind)
```

Comment:
IMO this is also a bit weird.

`{"msg":"Attempting to fetch","deployed":"Service"}"

I was thinking of:

```suggestion
	log.Info("Attempting to fetch deployed resource", "kind", kind)
```

Producing:

`{"msg":"Attempting to fetch deployed resource","kind":"Service"}"

Wdyt?